tests/e2e/core-tests/specs/merchant/wp-admin-product-import-csv.test.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+ const {
+	merchant,
+	setCheckbox,
+	moveAllItemsToTrash
+} = require( '@woocommerce/e2e-utils' );
+const getCoreTestsRoot = require( '../../core-tests-root' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe
+} = require( '@jest/globals' );
+
+const path = require( 'path' );
+const coreTestsPath = getCoreTestsRoot();
+const filePath = path.resolve( coreTestsPath, 'sample-data/sample_products.csv' );
```

Comment:
With previous comment:
```
const filePath = path.resolve( coreTestsPath.appRoot, 'sample-data/sample_products.csv' );
const filePathOverride = path.resolve( coreTestsPath.packageRoot, '/test-data/sample_products_override.csv' );
```

I did not test that. It may need some adjustments.

Reply:
I did and it worked 👍 @rrennick

Reply:
@veljkho Looks good. This worked great for me locally.